root/usr/share/nethesis/NethServer/Language/en/NethServer_Module_Group.php

```diff
@@ -14,3 +14,5 @@
 $L['groupname_label'] = 'Group name'; 
 $L['valid_platform,group-create,group-length,1'] = 'Name is too long: ${reason}';
 $L['group_exists'] = 'The group already exists';
+$L['CreatePseudoRecords_label'] = 'Create also a shared mailbox and an email address named after the group';
```

Comment:
What about removing "also"?

    Create a shared mailbox and an email address named after the group